### ISSUE TITLE
vertex color baking

### ISSUE THREAD

**Issue by ScanMountGoat** [ssbucarlos/smash-ultimate-blender#181]
There isn't any documentation on vertex color baking. A lot of custom stages lack edited vertex color, which can dramatically improve the visual quality. There should be some guidelines on how to account for the vertex color scale. One approach is to just add a page on the wiki for baking to color sets. This shouldn't require any additional code on our end unless there isn't a straightforward way to rescale vertex color while baking.

**Comment by ssbucarlos** [comment]
yeah the existing vertex color stuff i wrote down is super barebones and pretty much only tells the user to go look at other documentation. Some more detailed documentation about how to actually make the custom vertex colors in blender would be nice

**Comment by ssbucarlos** [comment]
I added a wiki page for editing vertex colors + added an operator for setting the vertex colors exactly.
Its still missing the instructions on baking from cycles to the colorSet1
https://github.com/ssbucarlos/smash-ultimate-blender/wiki/Editing-Vertex-Colors

**Comment by ScanMountGoat** [comment]
Thanks for adding that. I looked over the wiki to fix some small typos and other inconsistencies. We probably don't need to be super in depth on how to bake vertex colors. It looks like these instructions still work on 4.0. https://blender.stackexchange.com/questions/271985/how-to-bake-texture-to-vertex-colors
